src/platform/packages/shared/kbn-esql-validation-autocomplete/src/validation/function_validation.ts

```diff
@@ -487,7 +488,7 @@ function validateFunctionColumnArg(
   parentCommand: string
 ) {
   const messages: ESQLMessage[] = [];
-  if (!(isColumnItem(actualArg) || isIdentifier(actualArg))) {
+  if (!(isColumnItem(actualArg) || isIdentifier(actualArg)) || isParametrized(actualArg)) {
```

Comment:
`isVariable`? I'm just trying to figure out whether we want these called parameters or variables. Definitely a +1 to https://github.com/elastic/kibana/issues/211681

Reply:
Agreed, just used the existing helper here